wallet/manager.go

```diff
@@ -34,33 +65,39 @@ func (w *Manager) GetValidatorAddress(
 }
 
 func (w *Manager) CreateWallet(
-	mnemonic, language,
-	password, walletPath string,
-) error {
-	wlt, err := Create(walletPath, mnemonic, language, w.chainType)
+	password, walletName string,
```

Comment:
Parameters order can be like `walletName, password string,` to be almost same as Restore function